projects/libgit2/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build project
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX="$WORK" \
+      -DBUILD_SHARED_LIBS=OFF \
+      -DBUILD_CLAR=OFF \
+      -DUSE_HTTPS=OFF \
+      -DUSE_SSH=OFF \
+      -DUSE_BUNDLED_ZLIB=ON \
+
+make -j$(nproc)
+make install
+
+$CXX $CXXFLAGS -std=c++11 -I"$WORK/include" \
+    /src/fuzz_download_refs.cc -o $OUT/download_refs \
```

Comment:
Instead of fuzz_download_refs, better to call it download_refs_fuzzer for both filename and fuzzer name.